### PR TITLE
HCAL SiPMParameters, geometry 2017/2018, tracking MVA 2017/2018

### PR DESCRIPTION
**Pull request by franzoni**
see below: https://github.com/cms-sw/cmssw/pull/17530#issuecomment-280882838
an updated detailed description of the updates to the global tags of autoCond.py

### CODE REVIEW

**Comment by franzoni** [comment]
please test

**Comment by abdoulline** [comment]
@kpedro88    Kevin,  AddOn 
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17530/17808/addOnTests/logs/cmsDriver-hlt_mc_Fake_cmsDriver.py_TTbar_Tauola_8TeV_TuneCUETP8M1_cfi_-s_GEN,SIM,DIGI,L1,DIGI2RAW__--mc_--scenario=pp_-n_10_--conditions_auto:run1_mc_Fake_--relval_9000,50_.log

fails with quite enigmatic message:

cmsRun: /build/cmsbld/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/cfff335e48ef2bf88849dadb4ca06724/opt/cmssw/slc6_amd64_gcc530/cms/cmssw/CMSSW_9_0_X_2017-02-14-2300/src/SimCalorimetry/HcalSimAlgos/src/HcalSiPM.cc:145: void HcalSiPM::setNCells(int): Assertion `nCells>0' failed.

I have the impression "deja vu" though. But cannot recall... Like something is set for SiPMs simulation, but no SiPMs channels specified...

**Comment by Martin-Grunewald** [comment]
This is a run-1 workflow using a run-1 GT. Perhaps the hcal code lost that capability?

**Comment by mmusich** [comment]
Hi @franzoni,
may I ask what is the rationale for this change:

 >  * **RunII Prompt processing** : [90X_dataRun2_PromptLike_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v3) as [90X_dataRun2_PromptLike_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_PromptLike_v3/90X_dataRun2_PromptLike_v2):
>      * ...
>      * `L1TUtmTriggerMenu` from O2O http://kkotov.github.io/l1o2o/talks/2016.11.28/#2 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2259/1.html


? 

the autoCond key `auto:run2_data_promptlike` (originally meant to be transient) is used currently only for 2016H relvals in the step [`RECODR2_2016reHLT_Prompt`](https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/Configuration/PyReleaseValidation/python/relval_steps.py#L1271-L1278) for these [workflows](https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/Configuration/PyReleaseValidation/python/relval_standard.py#L209-L219)

For these workflows the L1 menu is customized to be compatible with the current version of the  [`@relval2016` autoHLT key](https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/Configuration/HLT/python/autoHLT.py#L10). 
In summary, why use in the RECO Global Tag a different L1 menu than the one used in the HLT step (from a mere technical standpoint it still runs, even if I haven't check if it affects anywhere downstream objects/products).

**Comment by franzoni** [comment]
Sharing info about the crash (no solution yet):

. the standard ttbar run1 workflow  `runTheMatrix.py --what standard  -l 25 --command='-n 100'`   runs normally using  auto:run1_mc=90X_mcRun1_realistic_v3

. the add on test ( see:  https://github.com/cms-sw/cmssw/pull/17530#issuecomment-280244010 ) uses auto:[run1_mc_Fake]( https://github.com/cms-sw/cmssw/blob/48469124b3b5dd9cdd61c9a25b74065cba7a849f/Configuration/HLT/python/autoCondHLT.py#L32 ) which modifies the L1 menu

. the crash of the addon tests develops in the L1 step; the three steps alone `cmsDriver.py TTbar_Tauola_8TeV_TuneCUETP8M1_cfi -s GEN,SIM,DIGI` don't crash

@Martin-Grunewald can you comment on the nature of l1Menus['Fake']  ?

**Comment by Martin-Grunewald** [comment]
The L1T menu is a proper L1T menu (Run-I legacy L1T), likely the default in the actual GT anyway!
The "fake" refers to the HLT menu, which is a basic, stripped-down menu with some trivial paths only.

**Comment by kpedro88** [comment]
@salavat the DetId causing the failed assert is:
1195912256 -> (HO 11,64)

Can you check that the conditions have the proper number of pixels filled in for HO cells that use SiPMs?

**Comment by abdoulline** [comment]
@kpedro88

Kevin, this is HPD HO in Run 1 (SiPMs type = 7) 
https://twiki.cern.ch/twiki/pub/CMS/HcalSiPMParameters/HcalSiPMParameters_RunI-2017compatibility.txt

And the most enigmatic is why this happens if regular Digitization is OK...

**Comment by kpedro88** [comment]
@abdoulline then maybe we need to check the MCParams? Somehow the vector `theHOSiPMDetIds` in `HcalDigitizer` is not empty.

**Comment by abdoulline** [comment]
Kevin, you're right,  (HO 11, 64) is set to shapeId 202  in 
HcalMCParams_v3.0_mc (Run 1 MC).
So there is a discrepancy between MCParams in that sense and SiPMsParams 
(type = 7 HPD) for this channels... Need to check other ones as well 
(tomorrow early morning)
Do you think this might be the culprit?

**Comment by kpedro88** [comment]
Almost certainly... we should make the tags consistent (probably update SiPMParameters if the MCParams is accurate)

**Comment by franzoni** [comment]
@kpedro88 @abdoulline 
once you'll have cleared the picture, can you help understand why the assert happens only in the specific combination of the addOn test and not in the regular run1 relvals ?

I'll be mostly out of office tomorrow (Friday). Please let me know here what is needed from AlCa/Db to get this PR approvable, I'll try to do asap. Having this PR merged is essential to have a simpler/clean integration of the plan1 geometry+conditions you're preparing, as we've agreed via email.

Thanks!

**Comment by kpedro88** [comment]
@franzoni it could just be a random number effect... the SiPM simulation is only run for a cell if there is a SimHit in that cell.

**Comment by abdoulline** [comment]
Right, SiPMParameters to correspond to HcalMCParams_v3.0_mc (there might 
be something changed later in Run 1, which served as a "template" for 
SiPMParameters) 
I'll send a request to Maxim/Walter...

**Comment by abdoulline** [comment]
well may be, as HO doesn't take a real signal quite often...

**Comment by franzoni** [comment]
@abdoulline 
> I'll send a request to Maxim/Walter...
are you suggesting there's new tags coming ?

**Comment by abdoulline** [comment]
Yes, tomorrow...
HcalSiPMParameters_v1.1_mc -> HcalSiPMParameters_v1.2_mc

**Comment by abdoulline** [comment]
Run 1 - related fixes/updates have just been submitted for HCAL SiPMParameters :
https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html

**Comment by franzoni** [comment]
Thanks @abdoulline
I'll reintegrate
By Monday at latest

**Comment by slava77** [comment]
about significance of the failure: in my tests with a few more run1 workflows, I see about a half of my "short" run1 runTheMatrix failing. 
So, it's just small numbers statistics.

**Comment by franzoni** [comment]
Thanks to the new tags form Walter et al:
https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html

the addOn test `TTbar_Tauola_8TeV_TuneCUETP8M1_cfi -s GEN,SIM,DIGI,L1,DIGI2RAW --mc --scenario=pp -n 10 --conditions auto:run1_mc_Fake --relval 9000,50 --datatier GEN-SIM-RAW --eventcontent RAWSIM --customise=HLTrigger/Configuration/CustomConfigs.L1T --fileout file:RelVal_Raw_Fake_MC.root` which failed reproducibly at event 5,
worked ok for 500.

tags integrated in queues, GT's update in this pr by the end of the day

**Comment by franzoni** [comment]
please test

**Comment by franzoni** [comment]
# Summary of changes in Global Tags

## RunI simulation

   * **RunI Heavy Ions scenario** : [90X_mcRun1_HeavyIon_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun1_HeavyIon_v4) as [90X_mcRun1_HeavyIon_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun1_HeavyIon_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun1_HeavyIon_v4/90X_mcRun1_HeavyIon_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html

   * **RunI Ideal scenario** : [90X_mcRun1_design_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun1_design_v4) as [90X_mcRun1_design_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun1_design_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun1_design_v4/90X_mcRun1_design_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html

   * **RunI Proton-Lead scenario** : [90X_mcRun1_pA_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun1_pA_v4) as [90X_mcRun1_pA_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun1_pA_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun1_pA_v4/90X_mcRun1_pA_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html

   * **RunI Realistic scenario** : [90X_mcRun1_realistic_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun1_realistic_v4) as [90X_mcRun1_realistic_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun1_realistic_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun1_realistic_v4/90X_mcRun1_realistic_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html

## RunII simulation

   * **RunII Asymptotic scenario** : [90X_mcRun2_asymptotic_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_asymptotic_v4) as [90X_mcRun2_asymptotic_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_asymptotic_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun2_asymptotic_v4/90X_mcRun2_asymptotic_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html


   * **RunII CRAFT scenario** : [90X_mcRun2cosmics_startup_deco_v1](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2cosmics_startup_deco_v1) as [90X_mcRun2cosmics_startup_peak_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2cosmics_startup_peak_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun2cosmics_startup_deco_v1/90X_mcRun2cosmics_startup_peak_v2):
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html
      * change the noise from PEAK to DECO

   * **RunII Heavy Ions scenario** : [90X_mcRun2_HeavyIon_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_HeavyIon_v4) as [90X_mcRun2_HeavyIon_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_HeavyIon_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun2_HeavyIon_v4/90X_mcRun2_HeavyIon_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html

   * **RunII Ideal scenario** : [90X_mcRun2_design_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_design_v4) as [90X_mcRun2_design_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_design_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun2_design_v4/90X_mcRun2_design_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html

   * **RunII Proton-Lead scenario** : [90X_mcRun2_pA_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_pA_v5) as [90X_mcRun2_pA_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_pA_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun2_pA_v5/90X_mcRun2_pA_v3):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html

   * **RunII Startup scenario** : [90X_mcRun2_startup_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_startup_v4) as [90X_mcRun2_startup_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_mcRun2_startup_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_mcRun2_startup_v4/90X_mcRun2_startup_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html

## RunI data

   * **RunI HLT processing** : [90X_dataRun2_HLT_frozen_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v2) as [90X_dataRun2_HLT_frozen_v0](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v0) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_frozen_v2/90X_dataRun2_HLT_frozen_v0):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

   * **RunI Offline processing** : [90X_dataRun2_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v4) as [90X_dataRun2_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_v4/90X_dataRun2_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

## RunII data

   * **RunII HLTHI processing** : [90X_dataRun2_HLTHI_frozen_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v2) as [90X_dataRun2_HLTHI_frozen_v0](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v0) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLTHI_frozen_v2/90X_dataRun2_HLTHI_frozen_v0):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

   * **RunII HLT relval processing** : [90X_dataRun2_HLT_relval_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v2) as [90X_dataRun2_HLT_relval_v0](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v0) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_relval_v2/90X_dataRun2_HLT_relval_v0):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

   * **RunII Offline processing** : [90X_dataRun2_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v3) as [90X_dataRun2_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_v3/90X_dataRun2_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

   * **RunII Offline relval processing** : [90X_dataRun2_relval_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_relval_v4) as [90X_dataRun2_relval_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_relval_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_relval_v4/90X_dataRun2_relval_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

   * **RunII Prompt processing** : [90X_dataRun2_PromptLike_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v4) as [90X_dataRun2_PromptLike_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_PromptLike_v4/90X_dataRun2_PromptLike_v2):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

   * **RunI HLT processing** : [90X_dataRun2_HLT_frozen_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v2) as [90X_dataRun2_HLT_frozen_v0](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v0) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_frozen_v2/90X_dataRun2_HLT_frozen_v0):
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

## Upgrade

   * **PhaseI 2017 cosmics scenario** : [90X_upgrade2017cosmics_realistic_peak_v11](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v11) as [90X_upgrade2017cosmics_realistic_peak_v9](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v9) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v11/90X_upgrade2017cosmics_realistic_peak_v9):
      * geometry update https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2818.html
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html

   * **PhaseI 2017 design scenario** : [90X_upgrade2017_design_IdealBS_v11](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v11) as [90X_upgrade2017_design_IdealBS_v9](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v9) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_design_IdealBS_v11/90X_upgrade2017_design_IdealBS_v9):
      * geometry update https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2818.html
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

   * **PhaseI 2017 realistic scenario** : [90X_upgrade2017_realistic_v11](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v11) as [90X_upgrade2017_realistic_v9](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v9) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_realistic_v11/90X_upgrade2017_realistic_v9):
      * geometry update https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2818.html
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

   * **PhaseI 2018 cosmics scenario** : [90X_upgrade2018cosmics_realistic_peak_v12](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_peak_v12) as [90X_upgrade2018cosmics_realistic_peak_v10](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_peak_v10) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018cosmics_realistic_peak_v12/90X_upgrade2018cosmics_realistic_peak_v10):
      * geometry update https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2818.html
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html


   * **PhaseI 2018 design scenario** : [90X_upgrade2018_design_IdealBS_v12](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v12) as [90X_upgrade2018_design_IdealBS_v10](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v10) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_design_IdealBS_v12/90X_upgrade2018_design_IdealBS_v10):
      * geometry update https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2818.html
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html 

   * **PhaseI 2018 realistic scenario** : [90X_upgrade2018_realistic_v12](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v12) as [90X_upgrade2018_realistic_v10](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v10) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_realistic_v12/90X_upgrade2018_realistic_v10):
      * geometry update https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2818.html
      * Updates to HCAL SiPMParameters https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2808/1/1/1/1/1/1/1/3/1/1.html
      * Updates to Hcal RespCorrs to include isotrack https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2823.html
      * GBRForest tags for phase1 tracking MVA selection , see https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2820.html

**Comment by franzoni** [comment]
@mmusich thanks, I've addressed your comment.

**Comment by franzoni** [comment]
@mariadalfonso  ( https://github.com/cms-sw/cmssw/pull/17533 ) @makortel ( https://github.com/cms-sw/cmssw/pull/17537  )
you may want to consider the extra commit for your pull requests,
the crash with the run1 addon test is addressed by new payloads

**Comment by franzoni** [comment]
comparison to the baseline of the short matrix:
    * no changes in runI mc workflow, where expected (statistics?)
    * no changes in runI data workflow, where expected  (statistics?)
    * no changes in runII data workflow, where expected  (statistics?)
    * 2017 workflows: differences in most detector objects, expected from geometry change
    * 2017 workflows: differences in most detector objects, expected from geometry change

**Comment by franzoni** [comment]
+1

**Comment by abdoulline** [comment]
Giovanni, from HCAL side no changes expected indeed, as we've moved HPD 
constants from config to DB...

**Comment by makortel** [comment]
@davidlange6 @slava77

Is merging this first and rebasing #17533 and #17537 on top of an IB (my preference) still viable for pre5?

**Comment by davidlange6** [comment]
+1
